solr/core/src/java/org/apache/solr/core/CachingDirectoryFactory.java

```diff
@@ -405,11 +398,6 @@ public void incRef(Directory directory) {
 
   @Override
   public void init(@SuppressWarnings("rawtypes") NamedList args) {
-    maxWriteMBPerSecFlush = (Double) args.get("maxWriteMBPerSecFlush");
-    maxWriteMBPerSecMerge = (Double) args.get("maxWriteMBPerSecMerge");
-    maxWriteMBPerSecRead = (Double) args.get("maxWriteMBPerSecRead");
-    maxWriteMBPerSecDefault = (Double) args.get("maxWriteMBPerSecDefault");
-
```

Comment:
Why is this all deleted?

Reply:
I simply noticed that those fields were private and actually never used. I think they should be removed for clarity, but that's not necessary for this PR.

Reply:
I looked at these a week ago.  It's lingering left-over stuff from a refactoring McCandless did years ago in where some thresholds are configured.